### ISSUE TITLE
[Crop & Soil] Enable complex combinations of harvesting operations

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#1825]
Grooming session should determine if this is too complex and issue and should be broken down further.

- Analyze the current implementation of harvesting operations and identify limitations in supporting complex combinations.
- Redesign the harvesting system to allow for multiple harvesting operations to be performed on a single crop, such as harvesting a portion of the crop as silage and another portion as grain.
- Implement the necessary in the CropManagement and HarvestOperations classes to support complex harvesting scenarios.
- Update the user interface and reporting functionality to show the different types of yields obtained from a single crop.

**Comment by ew3361zh** [comment]
Grooming session - @ew3361zh will follow up with SMEs on prioritizing this issue.

**Comment by ew3361zh** [comment]
@KFosterReed could you take a look at this issue with a similar approach you took to the other C&S issues? i.e. how feasible is this right now and how high of a priority would you say it is?

**Comment by KFosterReed** [comment]
I would say this is a low priority and needs SME input before it is achievable for dev team.

Looking at the current crop classes where each crop class is defined, in part, by the primary product being harvested from the plant (`CornGrain`) for example, I can think of at least two options for how to approach multiple products from the harvest of one plant:

- Add new classes that are specific for each multiple harvest practice (i.e. `WheatGrainAndStraw`)
- Create a method that allows for a secondary product to be harvested in addition to the primary product that is represented in the crop class.

**Comment by matthew7838** [comment]
Grooming session: Still waiting on SME input.

**Comment by allisterakun** [comment]
Grooming session: moving this issue to blocked as we wait for the SME input.

**Comment by Isgotkowitz** [comment]
> Grooming session: Still waiting on SME input.

Yes it is

**Comment by KFosterReed** [comment]
SME Grooming Session: I believe we have the SME support needed for this issue now but am marking it as a low priority for now
